src/uu/dircolors/src/dircolors.rs

```diff
@@ -52,28 +64,78 @@ pub fn guess_syntax() -> OutputFmt {
     }
 }
 
+fn get_usage() -> String {
+    format!("{0} {1}", executable!(), SYNTAX)
+}
+
 pub fn uumain(args: impl uucore::Args) -> i32 {
     let args = args
         .collect_str(InvalidEncodingHandling::Ignore)
         .accept_any();
 
-    let matches = app!(SYNTAX, SUMMARY, LONG_HELP)
-        .optflag("b", "sh", "output Bourne shell code to set LS_COLORS")
-        .optflag(
-            "",
-            "bourne-shell",
-            "output Bourne shell code to set LS_COLORS",
+    let usage = get_usage();
+
+    /* Clap has .visible_alias, but it generates help like this
+     *     -b, --sh                output Bourne shell code to set LS_COLORS [aliases: bourne-shell]
+     * whereas we want help like this
+     *     -b, --sh                output Bourne shell code to set LS_COLORS
+     *         --bourne-shell      output Bourne shell code to set LS_COLORS
+     * (or preferably like the original, but that doesn't seem possible with clap:)
+     *     -b, --sh, --bourne-shell    output Bourne shell code to set LS_COLORS
+     * therefore, command aliases are defined manually as multiple commands
+     */
+    let matches = App::new(executable!())
+        .version(crate_version!())
```

Comment:
Standardising this would be great! Personally, I prefer `crate_version!`.

Reply:
yeah, ditto

Reply:
done in https://github.com/uutils/coreutils/pull/2334/files

Reply:
Cool.  I kept `crate_version!()` here.